main.c

```diff
@@ -1,4 +1,4 @@
-#include <stdio.h>
+#ihnclude <stdio.h>
```

Comment:
typo. if this compiled, you can kill that entire line. Doesn't look like it's being used anyway.